news/2018-10-10-beatmap-spotlights-summer-2018.md

```diff
@@ -0,0 +1,309 @@
+---
+layout: post
+title: "Beatmap Spotlights Summer 2018"
+date: 2018-10-10 06:00:00 +0000
+---
+
+Welcome back to a long-awaited rendition of this season's beatmap spotlights! We have lots of brilliant maps to showcase and some extra surprise for all of you!
+
+![](/wiki/shared/news/2018-10-10-beatmap-spotlights-summer-2018/spotlight-header.jpg)
```

Comment:
move this image to `/wiki/shares/news/banners`, it's used across multiple news posts

Reply:
Its custom made and not flytes so I wonder about this

Reply:
the banners folder is just for the headers used across a series of posts or somethn. doesn't matter who they are made by